matchms/exporting/metadata_export.py

```diff
@@ -90,8 +90,8 @@ def export_metadata_as_csv(
 
 
 def _subset_metadata(
-    include_fields: List[str], metadata: np.array, columns: Set[str]
-) -> Tuple[np.array, Set[str]]:
+    include_fields: list[str], metadata: np.array, columns: list[str]
```

Comment:
other type hints in this file are using `typing`'s _List_, _Dict_ ,.... .I guess we should switch to built-in types _list_ for the other methods to make it more consistent. But maybe in a different PR?

Reply:
Using type hints like this is only supported for python 3.10+. But I think other things already break compatibility with 3.9, so I don't think it is okay indeed to switch towards this other form of type hinting.

Reply:
Yes, matchms requires ">=3.10,<3.14". I'll quickly add this.